app/channels/mentor_request_channel.rb

```diff
@@ -9,10 +9,8 @@ def unsubscribed; end
 
   def self.broadcast!(request)
     broadcast_to request, {
-      mentor_request: {
```

Comment:
I've simplified the structure. Within the scope of the channel, the extra nesting is unnecessary. It's already clear that we're dealing with a fragment of a mentor request.

Reply:
I'm not sure about this. We may in the future want to have something like broadcasting meta info, e.g. what's changed or something, and that shouldn't be part of `mentor_request`. I prefer to have it nested.

Reply:
okay